### PR TITLE
Implement infrastructure for sending emails

### PR DESCRIPTION
**Pull request by connorcimowsky**
Confirmation emails to come in a later PR.

### CODE REVIEW

File: src-django/requirements.txt [hunk 1]
```diff
@@ -14,3 +14,5 @@ flake8==2.5.1
 gunicorn==19.4.5
 nose==1.3.7
 psycopg2==2.6.1
+celery==3.1.18
```
Comment (Kapin) [inline src-django/requirements.txt]: nit: alphabetize plz

File: src-django/sanaprotocolbuilder/celery.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import
+
+import os
+
+from celery import Celery
+
+# set the default Django settings module for the 'celery' program.
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'sanaprotocolbuilder.settings')
+
+from django.conf import settings  # noqa
```
Comment (Kapin) [inline src-django/sanaprotocolbuilder/celery.py]: Why does this need a noqa?

File: src-django/sanaprotocolbuilder/celery.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+# Using a string here means the worker will not have to
+# pickle the object when using Windows.
+app.config_from_object('django.conf:settings')
+app.autodiscover_tasks(lambda: settings.INSTALLED_APPS)
+
+
+@app.task(bind=True)
+def debug_task(self):
+    print('Request: {0!r}'.format(self.request))
```
Comment (Kapin) [inline src-django/sanaprotocolbuilder/celery.py]: print shouldn't have brackets

File: src-django/sanaprotocolbuilder/celery.py (hunk 2 above)
Comment (connorcimowsky) [inline src-django/sanaprotocolbuilder/celery.py]: http://docs.celeryproject.org/en/latest/django/first-steps-with-django.html

**Comment by Trinovantes** [comment]
+1

**Comment by Ommy** [comment]
LGTM

**Pull request merged**